### ISSUE TITLE
Date field causing internal server error

### ISSUE THREAD

**Issue by stevefloat** [verbb/formie#402]
**Issue**
When using a Date field with a display type of 'dropdown', if you select "None" as the 'Default value' the form causes an Internal Server Error on the frontend and on the submission screens (admin/formie/submissions).

See line from the web.log:
`2021-03-16 14:49:30 [-][-][8f8mhia7dga1th4a3is7bp3v0u][error][Error] Error: Call to a member function format() on array in C:\xxx\xxx\xxx\vendor\verbb\formie\src\fields\formfields\Date.php:206`

**To resolve**
Setting 'Default value' to "Today's date/time" resolves the error.

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.22
- Craft version: 3.5.13.2 (PRO)
- Multi-site: No

**Additional context**
Error can be reproduced in dev and production environments and on both Linux and Windows servers

Whilst I've instructed the website administrators to ensure they set a default value here, it would be good if the error was caught before throwing an Internal Server error.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.23",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.24](https://github.com/verbb/formie/releases/tag/1.3.24)

**Comment by stevefloat** [comment]
Great - thanks @engram-design!
